src/_articles/dart-vm/numeric-computation.md

```diff
@@ -1,13 +1,26 @@
 ---
 title: Numeric Computation
 description: How you store and use numbers can have a big impact on your app's performance. This article focuses on the Dart VM.
-date: 2013-05-22
+original-date: 2013-05-22
+date: 2018-07-26
 category: dart-vm
 obsolete: true
 ---
 
 _Written by John Mccutchan <br>
-May 2013_
+May 2013 (note added July 2018)_
+
+<aside class="alert alert-warning" markdown="1">
+  In Dart 2, "proper" integers were replaced with 64-bit signed, wrapping
```

Comment:
"Proper integers" is my perspective. A better term for a general audience is "arbitrary-precision integers".

Reply:
Thanks, that's better. :)